receiver/podmanreceiver/factory.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	typeStr           = "podman_stats"
 	defaultAPIVersion = "3.3.1"
+	defaultMaxRetries = "10"
```

Comment:
I think default should probably be forever or a very large number. Given the current exponential timeoff implementation, how long would it take before 10 attempts are exhausted?

Reply:
As discussed on the slack channel that we want to continuously retry forever and not restrict the number of retries to a specific value. So, removing this parameter from the code.